components/Snackbar/src/MDCSnackbarMessageView.m

```diff
@@ -349,17 +349,20 @@ - (instancetype)initWithMessage:(MDCSnackbarMessage *)message
                                            [[self class] bundle],
                                            @"Dismissal accessibility hint for Snackbar");
 
-    // For VoiceOver purposes, the label is the primary 'button' for dismissing the Snackbar, so
-    // we'll make sure the label looks like a button.
+    // For UIAccessibility purposes, the label is the primary 'button' for dismissing the Snackbar,
+    // so we'll make sure the label is treated like a button.
     _label.accessibilityTraits = UIAccessibilityTraitButton;
     _label.accessibilityIdentifier = MDCSnackbarMessageTitleAutomationIdentifier;
     _label.accessibilityHint = accessibilityHint;
 
-    // If an accessibility label was set on the message model object, use that instead of the text
-    // in the label.
+    // If an accessibility label or hint was set on the message model object, use that instead of
+    // the text in the label or the default hint.
     if ([message.accessibilityLabel length]) {
```

Comment:
[`length` is an NSUInteger](https://developer.apple.com/documentation/foundation/nsstring/1414212-length?language=objc), so I don't think it's ever anything other than 0 or the actual length. In our ObjC code, I think we do a lot of conditional checking for nil by implicit casting to a `BOOL` and this is very similar. If you think it's important, I can change it but I was keeping with the file's approach and trying to minimize changes.

Reply:
by doing some quick searches it looks like we've been inconsistent with the way we check for `count`, but not for `length`, so i'm leaving this as you see fit. This would be a good coding style question though.